packages/forms-web-app/src/views/layouts/main.njk

```diff
@@ -38,15 +40,17 @@
         }) }}
       {% endblock %}
 
-      {% block backButton %}
-        {{ govukBackLink({
-          text: "Back",
-          href: navigation[1],
-          attributes: {
-            'data-cy': 'back'
-          }
-        }) }}
-      {% endblock %}
+      {% if showBackLink %}
+        {% block backButton %}
+          {{ govukBackLink({
+            text: "Back",
+            href: navigation[1],
+            attributes: {
+              'data-cy': 'back'
+            }
+          }) }}
+        {% endblock %}
+      {% endif %}
```

Comment:
If a block doesn't need to be displayed in a template then an empty block can be added to that template as it will override the default functionality of the block definition, so we don't need to add a conditional to display/hide it.

So for example if you add the following to `granted-or-refused-permission-out.njk` then the back button will be hidden

```
{% block backButton %}
{% endblock %}
```